openslides/core/projector.py

```diff
@@ -19,12 +19,23 @@ def check_data(self):
             raise ProjectorException('Custom slide does not exist.')
 
     def get_requirements(self, config_entry):
+        from openslides.agenda.views import ItemViewSet
         pk = config_entry.get('id')
         if pk is not None:
+            slide = CustomSlide.objects.get(pk=pk)
```

Comment:
Please use the same style as in all other slide (catch DoesNotExist exception ...).